### PR TITLE
Build and publish with Whiskey

### PR DESCRIPTION
**Pull request by decoyjoe**
Updates to get the repository building and publishing with Whiskey.

### CODE REVIEW

File: Tests/Initialize-BitbucketServerAutomationTest.ps1 [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # limitations under the License.
 
 & (Join-Path -Path $PSScriptRoot -ChildPath '..\BitbucketServerAutomation\Import-BitbucketServerAutomation.ps1' -Resolve)
-& (Join-Path -Path $PSScriptRoot -ChildPath '..\Carbon\Import-Carbon.ps1' -Resolve)
+& (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\Carbon\*\Import-Carbon.ps1' -Resolve)
```
Comment (splatteredbits) [inline Tests/Initialize-BitbucketServerAutomationTest.ps1]: Change this to `Import-Module -Name (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\Carbon') -Force` so we don't have to worry about what version(s) might be installed and we'll always get the latest.

File: Tests/Set-BBServerCommitBuildStatus.Tests.ps1 [hunk 2]
```diff
@@ -14,7 +14,7 @@
 Set-StrictMode -Version 'Latest'
 
 & (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-BitbucketServerAutomationTest.ps1' -Resolve)
-& (Join-Path -Path $PSScriptRoot -ChildPath '..\LibGit2\Import-LibGit2.ps1' -Resolve)
+& (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\GitAutomation\*\Import-GitAutomation.ps1' -Resolve)
```
Comment (splatteredbits) [inline Tests/Set-BBServerCommitBuildStatus.Tests.ps1]: Change to `Import-Module -Path (Join-Path -Path $PSScriptRoot -ChildPath '..\PSModules\GitAutomation') -Force`.

File: Uninstall-BitbucketServer.ps1 [hunk 3]
```diff
@@ -12,26 +12,41 @@
...
 
-& (Join-Path -Path $PSScriptRoot -ChildPath '.\Carbon\Import-Carbon.ps1' -Resolve)
+& {
+    $VerbosePreference = 'SilentlyContinue'
 
-Get-Service -Name '*Bitbucket*' | 
+    $importCarbonModulePath = (Join-Path -Path $PSScriptRoot -ChildPath '.\PSModules\Carbon\2.6.0\Import-Carbon.ps1')
+    if (-not (Test-Path -Path $importCarbonModulePath -PathType Leaf))
+    {
+        Save-Module -Name 'Carbon' -Path (Join-Path -Path $PSScriptRoot -ChildPath 'PSModules') -RequiredVersion '2.6.0' -Force
```
Comment (splatteredbits) [inline Uninstall-BitbucketServer.ps1]: Replace this with a message to run `init.ps1` to install Carbon.

**Review by splatteredbits — CHANGES_REQUESTED** [review]

File: Uninstall-BitbucketServer.ps1 (hunk 3 above)
Comment (decoyjoe) [reply Uninstall-BitbucketServer.ps1]: Did you mean `build.ps1`? The init script doesn't install Carbon, the Whiskey build does.

Comment (splatteredbits) [reply Uninstall-BitbucketServer.ps1]: Yes. `build.ps1 -Initialize`.

File: Tests/Initialize-BitbucketServerAutomationTest.ps1 (hunk 1 above)
Comment (decoyjoe) [reply Tests/Initialize-BitbucketServerAutomationTest.ps1]: Curious, so when should we be calling `Import-Carbon.ps1` vs importing normally with the `Import-Module` cmdlet?

**Comment by decoyjoe** [comment]
@splatteredbits Requested changes made. Ready for re-review.

**Review by splatteredbits — APPROVED** [review]

**Pull request merged**